### PR TITLE
Built Navbar, HeroSection, EventsCard, ServiceCard, Footer, 404 Page

### PR DESCRIPTION
**Pull request by WeSecureYou**
I've built the following things mentioned in the title so far and currently building the rest of the things, expect it to be done within a week :))

### CODE REVIEW

**Comment by WeSecureYou** [comment]
Review it and let me know if any changes is required. If you want to see live demo of the work i have hosted it for temporary basis here: [Live Demo](https://websitecss.vercel.app/)

File: src/components/EC.jsx
Comment (kpatel0170) [inline src/components/EC.jsx:1]: Event's card doesn't show appropriate information, 
- date, time and location

File: src/components/EventsCard.jsx
Comment (kpatel0170) [inline src/components/EventsCard.jsx:1]: The data can be in separate file or maybe dynamic based on event name:

example: github-event.json or react-event.json

File: src/components/ServiceSection.jsx
Comment (kpatel0170) [inline src/components/ServiceSection.jsx:1]: separate data format

File: src/pages/Elections.jsx
Comment (kpatel0170) [inline src/pages/Elections.jsx:1]: elections are yearly so maybe with 2024.json or 2024.md

**Review by kpatel0170 — CHANGES_REQUESTED** [review]
- [x] separate static data (/data/...) - .md format or .json format
- [x] load data dynamically based on data files
- [x] add code of conduct page from https://github.com/csss-uofr/code-of-conduct
- [x] add constitution page 
- [x] update colors best suited based on current logo

**Comment by WeSecureYou** [comment]
Noted. I just need the current UR CSSS logo in the best quality possible. Thanks!

**Comment by WeSecureYou** [comment]
All of the above-mentioned changes have been taken care of. Please take a look at the [Live Demo](https://websitecss.vercel.app/) and let me know if anything further is required

**Comment by WeSecureYou** [comment]
Please note that the data in the JSON files is dummy data since I don't have access to the actual data. The About Us page might need some editing because I don't have the actual content that needs to be included on that page. The menu button on mobile devices still needs to be implemented (You can expect it to be done in the next few commits). Also, I'm not really sure what you meant by the 'Constitution Page'.

**Review by kpatel0170 — APPROVED** [review]
LGTM :)

**Pull request merged**